frontend/src/pages/AdminPage.js

```diff
@@ -1,168 +1,179 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Button from 'react-bootstrap/Button';
+import axios from "axios";
+
+axios.defaults.headers.common['User'] = process.env.REACT_APP_API_KEY
 
 const AddEmployee = () => {
-  return (
-    <div class="panel panel-primary">
-      <div class="panel-heading">
-        <h3 class="panel-title"><strong>Add Employee</strong></h3>
-      </div>
-      <div class="panel-body">
-        <form>
-          <div class="form-group">
-            <label for="employee-first-name-input">First Name</label>
-            <input class="form-control" id="employee-first-name-input" type="text"></input>
-          </div>
-          <div class="form-group">
-            <label for="employee-last-name-input">Last Name</label>
-            <input class="form-control" id="employee-last-name-input" type="text"></input>
-          </div>
-          <div class="form-group">
-            <label for="email-input">E-Mail</label>
-            <input class="form-control" id="email-input" type="text"></input>
-          </div>
-          <div class="form-group">
-            <label for="status-input">Employee Status (Y/N)</label>
-            <input class="form-control" id="status-input" type="text"></input>
-          </div>
-          <div className="col-md-4 offset-md-4">
-            <Button variant="primary" size="lg" block>
-                Submit
-           </Button>
+
+    const [formData, updateFormData] = React.useState([]);
+
+    const handleChange = (e) => {
+      updateFormData({
+        ...formData,
+        // Trimming any whitespace
+        [e.target.name]: e.target.value.trim()
+      });
+    };
+
+    var newEmployeeData = {
+        'firstName': formData["employee-first-name-input"],                     
+        'lastName': formData["employee-last-name-input"],
+        'email': formData["email-input"],
+        'FullName': formData["employee-first-name-input"] + formData["employee-last-name-input"],
```

Comment:
Nitpick:  can we follow one standard way of naming convention? all the fields are camel cased, but FullName is not.

Reply:
I'll make it consistent.